Vagrantfile

```diff
@@ -249,21 +251,19 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
             # create an interface for etcd cluster
             node.vm.network :private_network, ip: node_addr, virtualbox__intnet: "true", auto_config: false
             # create an interface for bridged network
-            if ENV['CONTIV_L3'] then
-              # create an interface for bridged network
-              node.vm.network :private_network, ip: "0.0.0.0", virtualbox__intnet: network_name, auto_config: false
-            else
-              node.vm.network :private_network, ip: "0.0.0.0", virtualbox__intnet: "true", auto_config: false
-            end
```

Comment:
This setting is required to set each each interface in a separate network. Had a discussion with @shaleman and @abhinandanpb regarding this as well; this could be the default and not be based on CONTIV_L3 check.